apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -493,6 +499,18 @@ process_request(#{<<"method">> := <<"channels.shutdown">>}, FsmPid) ->
     lager:warning("Channel WS: closing channel message received"),
     aesc_fsm:shutdown(FsmPid),
     no_reply;
+process_request(#{<<"method">> := <<"channels.close_solo">>}, FsmPid) ->
```

Comment:
Does this require update to `node/api/channel_ws_api.md` in protocol repo (after https://github.com/aeternity/protocol/pull/328 merged )?

Reply:
Yes, will update the documentation